app/views/forms/make_live/confirmation.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-l"><%= @form.name %></span>
+    <h1 class="govuk-heading-l">Your form is live</h1>
+
+    <p class="govuk-body">
+      The URL for your form is:
+
+      <span class="govuk-!-display-block govuk-!-margin-top-6 govuk-!-padding-bottom-6">
+        <%= govuk_link_to "#{ENV['RUNNER_BASE']}/form/#{@form.id}","#{ENV['RUNNER_BASE']}/form/#{@form.id}", { target:"_blank", rel:"noopener noreferrer"} %>
```

Comment:
You could optionally consider putting `#{ENV['RUNNER_BASE']}/form/#{@form.id}` in a variable somewhere to avoid the repetition.